config.js.example

```diff
@@ -22,5 +22,5 @@ global.ALLOWED_CLIENTS = [
   // W3C:
   /https?:\/\/(www\.)?w3c?\.org/i,
   // GitHub:
-  /https?:\/\/github\.com/i
+  /https?:\/\/.*\.github\.io/i
```

Comment:
(Come to think of it, and for the same reason, all these regex's should match the beginning of line too, eg `/^http…`. But, no! because in [`app.js#L120`](https://github.com/w3c/echidna/blob/master/app.js#L120) these are used to test *the origin header only*, and that shouldn't contain directories in it&hellip; So I think we're safe with this, @astorije. I'll merge tomorrow if you don't say otherwise.)